Pareto/AppHandlers.swift

```diff
@@ -335,7 +335,7 @@ class AppHandlers: NSObject, NetworkHandlerObserver {
     func copyLogs() {
         if !SystemUser.current.isAdmin {
             let alert = NSAlert()
-            alert.messageText = "Logs cannot be copied to the clipboard. Instead, you must use the Console app to capture the application logs by searching for “ParetoSecurity” and copying them to the clipboard."
+            alert.messageText = "Logs cannot be copied to the clipboard because you are using a non-admin macOS account. Instead, you must use the Console app to capture the application logs by searching for “Pareto Security” and copying them to the clipboard. See https://paretosecurity.com/docs/mac/logs for help."
```

Comment:
```suggestion
            alert.messageText = "Logs cannot be copied to the clipboard because you are using a non-admin macOS account. Instead, you must use the Console app to capture the application logs by searching for “Pareto Security” and copying them to the clipboard. See [our guide](https://paretosecurity.com/docs/mac/logs) for help."
```